### PR TITLE
Various group fixes

### PR DESCRIPTION
**Pull request by Bilb**

### CODE REVIEW

File: js/views/invite_friends_dialog_view.js [hunk 1]
```diff
@@ -14,11 +14,21 @@
...
+          d => !!d && d.isFriend() && d.isPrivate() && !d.isMe()
+        );
+      } else {
+        this.friends = convos.filter(
+          d =>
+            !!d &&
+            d.isFriend() &&
+            d.isPrivate() &&
+            !d.isMe() &&
+            !convo.get('members').includes(d.id)
```
Comment (neuroscr) [inline js/views/invite_friends_dialog_view.js:28]: won't this break invites for opengroups?

Comment (Bilb) [reply js/views/invite_friends_dialog_view.js:28]: this is in the NOT open group case of the `if`

Comment (neuroscr) [reply js/views/invite_friends_dialog_view.js:28]: ah my bad, sorry

Comment (Mikunj) [reply js/views/invite_friends_dialog_view.js:28]: Couldn't you also do
```javascript
this.friends = convos.filter(
          d => !!d && d.isFriend() && d.isPrivate() && !d.isMe()
);
if (!convo.isPublic()) {
   const members = convo.get('members') || [];
    this.friends = this.friends.filter(d => !members.includes(d.id);
}
```

Less duplication

**Pull request merged**